metadata.yaml

```diff
@@ -44,3 +44,6 @@ releaseSeries:
   - major: 1
     minor: 9
     contract: v1beta1
+  - major: 1
```

Comment:
this PR should update https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/test/e2e/data/shared/v1beta1_provider/metadata.yaml (this one is already updated in the main branch)

Reply:
🤦🏻 updated!